src/optimizer/join_order/join_node.cpp

```diff
@@ -6,55 +6,22 @@
 
 namespace duckdb {
 
-JoinNode::JoinNode(JoinRelationSet &set, const double base_cardinality)
-    : set(set), info(nullptr), has_filter(false), left(nullptr), right(nullptr), base_cardinality(base_cardinality) {
-	estimated_props = make_uniq<EstimatedProperties>(base_cardinality, 0);
+JoinNode::JoinNode(JoinRelationSet &set) : set(set), info(nullptr), left(nullptr), right(nullptr) {
```

Comment:
Again, a bit pedantic, but `optional_ptr` is `nullptr` by default, does not need to be initialized.